junit-platform-engine/src/main/java/org/junit/platform/engine/support/hierarchical/ForkJoinPoolHierarchicalTestExecutorService.java

```diff
@@ -57,15 +57,28 @@
 	 * @see DefaultParallelExecutionConfigurationStrategy
 	 */
 	public ForkJoinPoolHierarchicalTestExecutorService(ConfigurationParameters configurationParameters) {
-		forkJoinPool = createForkJoinPool(configurationParameters);
+		this(createConfiguration(configurationParameters));
+	}
+
+	/**
+	 * Create a new {@code ForkJoinPoolHierarchicalTestExecutorService} based on
+	 * the supplied {@link ParallelExecutionConfiguration}.
+	 *
+	 * @since 1.7
+	 */
+	public ForkJoinPoolHierarchicalTestExecutorService(ParallelExecutionConfiguration configuration) {
```

Comment:
```suggestion
	@API(status = EXPERIMENTAL, since = "1.7")
	public ForkJoinPoolHierarchicalTestExecutorService(ParallelExecutionConfiguration configuration) {
```